CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ When you're happy with the shape you've drawn, press <kbd>Finish</kbd>. You will
 	- **Website**: If you're describing a project, the project's main website would be suitable here.
 	- **Subreddit**: Format it like `r/subreddit`.
 	- **Discord**: Write the invite code, that the invite link without the `discord.gg/` part.
-	- **Wiki**: The page title of [The r/place Wiki](https://place-wiki.stefanocoding.me/).
+	- **Wiki**: The page title of [the r/place Wiki](https://place-wiki.stefanocoding.me/).
```

Comment:
I'm thinking that the brand name of the wiki is "The r/place Wiki" and not "r/place Wiki". The word "The" here is capitalized, so I think this should be reverted.

```suggestion
	- **Wiki**: The page title of [The r/place Wiki](https://place-wiki.stefanocoding.me/).
```

Reply:
Realistically, I don't think anybody includes the "The" in the name in general conversation. Including it and capitalizing it feels unnecessarily pompous, and also confusing.

Reply:
Yeah, I can see that. I would suggest for it to be renamed so it just "r/place Atlas" upon agreement of WT.